packages/example/pages/_app.tsx

```diff
@@ -287,6 +289,7 @@ function RainbowKitApp({
           ...demoAppInfo,
           ...(showDisclaimer && { disclaimer: DisclaimerDemo }),
         }}
+        ignoreChainModalOnConnect={ignoreChainModalOnConnect}
```

Comment:
If user doesn't want to have the chain modal before connection then they can provide `ignoreChainModalOnConnect` prop. This will sometimes help the UX since otherwise wallets have to connect + switch to the desired chain that they've choosen.

Some people are just interested to have the connect modal only which can prevent the switch chain modal prompt and connect to the wallet's current chain instead.